_pages/security-events.md

```diff
@@ -20,25 +25,35 @@ sidenav:
 
 ## Getting started
 
-login.gov allows partners to notify us of various security-related events through our API.
+login.gov allows partners and Relying Parties (RPs) to notify us of various security-related events through our API.
 
 ### Auto-discovery
 
 login.gov provides a JSON endpoint for OpenID Connect auto-discovery at `/.well-known/risc-configuration`. In our agency integration environment, this is available at <https://idp.int.identitysandbox.gov/.well-known/risc-configuration>
 
 ## Supported Events
 
-login.gov supports a subset of the [OpenID RISC Event Types][openid-risc-events]:
+login.gov custom events, based on the [OpenID RISC Event Types][openid-risc-events], but not specific ones from that list at this time.
+
+### Authorization Fraud Detected
+
+RPs should submit this event when they believe a user's credentials have been compromised, that somebody who is not the user
+was able to sign in to a user's account. login.gov may force the reset of user's password when we receive this event.
 
+The **event_type** for this is:
 
-### Account Credential Change Required
+```
+https://schemas.openid.net/secevent/risc/event-type/login-gov/authorization-fraud-detected
```

Comment:
So we're going to create new event types

1. the schemas.openid.net doesn't exist
2. I figured if I did a login.gov domain, people might expect it to exist, so I kept the old one
3. I added `/login-gov/` prefix to the event name, as a way to mark that it's ours?

Would love thoughts on if the actual domain of this should be login.gov

Reply:
Initial feeling is to have the URI be something that we _could_ make exist, even if it isn't necessary currently?

Reply:
Ok so more like:

```
https://schemas.login.gov/secevent/risc/event-type/authorization-fraud-detected
```

Reply:
Looks good to me, seems odd their schemas don't exist though.